src/shared/validators/maxLength.test.ts

```diff
@@ -0,0 +1,18 @@
+import { maxLength } from './maxLength';
+
+describe('Validator maxLength::', () => {
+  it('should return undefined if the string length is equal to or less than required', () => {
```

Comment:
the word `required` he is misleading, since we also have a `required` validator.
```suggestion
  it('should return undefined if the string length is equal to or less than maxLength parameter', () => {
```

Reply:
Done